metricbeat/module/etcd/leader/leader_test.go

```diff
@@ -31,15 +31,6 @@ import (
 	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
 )
 
-func TestEventMapping(t *testing.T) {
-	content, err := ioutil.ReadFile("../_meta/test/leaderstats.json")
-	assert.NoError(t, err)
-
-	event := eventMapping(content)
-
-	assert.Equal(t, event["leader"], string("924e2e83e93f2560"))
-}
-
```

Comment:
We don't need this test anymore?

Reply:
`eventMapping` function has changed - it loops over etcd followers and doesn't return anything.

Reply:
@gpop63 but then I cannot see how we test the new behaviour now, I might be missing something.

Reply:
Thank you very much @gpop63 !